src/AssemblySharedInfoGenerator/AssemblyInfoGenerator.csproj

```diff
@@ -74,4 +74,7 @@
   <Target Name="AfterBuild">
   </Target>
   -->
+  <Target Name="BeforeBuild">
+    <Exec Condition="'$(OS)'!='Unix'" Command="$(SolutionDir)restorepackages.bat" ContinueOnError="true" /> 
```

Comment:
DynamoRevit doesn't need to compile Mono so that we don't need the `Condition` here :)